src/main/java/pl/wrapper/parking/result/ErrorResult.java

```diff
@@ -0,0 +1,17 @@
+package pl.wrapper.parking.result;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Class {@code ErrorResult} represent exceptions.
+ * It contains error description and error code as Strings.
+ */
+@Getter
+@Setter
+@AllArgsConstructor
+public class ErrorResult {
```

Comment:
It's better to make it a record as the sole purpose of this class is carrying immutable data